### PR TITLE
disabling a range if it's beyond minDate or maxDate

### LINKED ISSUE #19: Last month doesn't adhere to min and max values?
I am using presets and have a preset called 'Last Month' that looks like this.
```ts
export const LAST_MONTH: DateRangePreset = {
  name: 'Last Month',
  range: [
    moment()
      .subtract(1, 'month')
      .startOf('month'),
    moment()
      .subtract(1, 'month')
      .endOf('month')
  ]
};
```
I have this construct for internal purposes and just send the `range` bit to ngx-daterangepicker-material.

This doesn't seem to be following the minDate and maxDate settings. I have another preset for `Last 30 Days` which seems to be working fine. 

```ts
export const LAST_30_DAYS: DateRangePreset = {
  name: 'Last 30 Days',
  range: [moment().subtract(30, 'days'), moment().subtract(1, 'days')]
};
```

### Last Month
image

### Last 30 Days
image

### PR DESCRIPTION
**Pull request by praveenpuglia**
Fixes #19 

Here, the `startDate` and `endDate` for "Last Month" range are outside the `minDate` and `maxDate` limits. So it's disabled.

## Screenshot
image

### CODE REVIEW

**Comment by praveenpuglia** [comment]
let me know once you merge and publish it to NPM. :)

File: src/daterangepicker/daterangepicker.component.ts [hunk 1]
```diff
@@ -717,4 +717,28 @@ export class DaterangepickerComponent implements OnInit {
...
+      let disabled = false;
+      const areBothBefore = rangeMarkers.every( date => {
+        return date.isBefore(this.minDate)
+      });
+
+      const areBothAfter = rangeMarkers.every( date => {
+        return date.isAfter(this.maxDate)
+      });
+
+      if(areBothBefore || areBothAfter) {
```
Comment (fetrarij) [inline src/daterangepicker/daterangepicker.component.ts]: could you remove  disabled variable and directly return the test?
````
const areBothBefore = rangeMarkers.every( date => {
        return date.isBefore(this.minDate)
      });

const areBothAfter = rangeMarkers.every( date => {
        return date.isAfter(this.maxDate)
     });
return (areBothBefore || areBothAfter;)

Comment (fetrarij) [reply src/daterangepicker/daterangepicker.component.ts]: @praveenpuglia just this and will merge :)

Comment (praveenpuglia) [reply src/daterangepicker/daterangepicker.component.ts]: Just committed the changes. review once :)

**Pull request merged**